components/activity-section.jsx

```diff
@@ -13,25 +13,33 @@ var ActivitySection = React.createClass({
           Section {this.props.title}
         </h2>
         {this.props.activities.map(function (activity, i) {
-          return (
-            <div className="activity" key={i}>
-              <Illustration height={122} width={122}
-              src1x={activity.image1x}
-              src2x={activity.image2x}
-              alt=""
-              link={activity.link}>
-                <div className="curriculum-description">
-                  <a href={activity.link}>
-                    <h3>{activity.title}</h3>
-                  </a>
-                  <p>
-                    <em>{activity.subtitle}</em>
-                  </p>
-                  <p className="description" dangerouslySetInnerHTML={{__html: activity.description}}></p>
-                </div>
-              </Illustration>
-            </div>
-          )
+          if (Object.keys(activity).length === 1 && activity.title) {
```

Comment:
Adding this to handle cases like this
![screen shot 2015-06-18 at 4 04 45 pm](https://cloud.githubusercontent.com/assets/2896608/8244281/dd8c07aa-15d3-11e5-8cc9-c9d2f42602e9.png)